### PR TITLE
#trivial Morelogging

### PR DESCRIPTION
**Pull request by JC1738**

### CODE REVIEW

**Comment by JC1738** [comment]
@nyarly wanted to get more logging into the cycle to see if help track down issue

**Comment by sous-machine-account** [comment]
LGTM

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: lib/resolver.go [hunk 1]
```diff
@@ -42,17 +43,21 @@ func (r *Resolver) queueDiffs(dcs *DeployableChans, results chan DiffResolution)
 	var wg sync.WaitGroup
 	for p := range dcs.Pairs {
 		sr := NewRectification(*p)
+		messages.ReportLogFieldsMessageWithIDs("Adding to queset", logging.ExtraDebug1Level, r.ls, p, sr)
```
Comment (samsalisbury) [inline lib/resolver.go]: Should prob fix this to "queueset" to help with filtering in Kibana.

File: lib/resolver.go [hunk 2]
```diff
@@ -42,17 +43,21 @@ func (r *Resolver) queueDiffs(dcs *DeployableChans, results chan DiffResolution)
...
 		queued, ok := r.QueueSet.PushIfEmpty(sr)
 		if !ok {
+			messages.ReportLogFieldsMessageWithIDs("failed to que", logging.ExtraDebug1Level, r.ls, p, sr)
```
Comment (samsalisbury) [inline lib/resolver.go]: Same here.

**Pull request merged**